registry/txt_test.go

```diff
@@ -897,8 +910,10 @@ func testTXTRegistryMissingRecordsWithPrefix(t *testing.T) {
 		Create: []*endpoint.Endpoint{
 			newEndpointWithOwner("oldformat.test-zone.example.org", "foo.loadbalancer.com", endpoint.RecordTypeCNAME, ""),
 			newEndpointWithOwner("txt.oldformat.test-zone.example.org", "\"heritage=external-dns,external-dns/owner=owner\"", endpoint.RecordTypeTXT, ""),
-			newEndpointWithOwner("oldformat2.test-zone.example.org", "bar.loadbalancer.com", endpoint.RecordTypeA, ""),
-			newEndpointWithOwner("txt.oldformat2.test-zone.example.org", "\"heritage=external-dns,external-dns/owner=owner\"", endpoint.RecordTypeTXT, ""),
+			newEndpointWithOwner("oldformat2.test-zone.example.org", "bar.loadbalancer.com", endpoint.RecordTypeA, "").WithSetIdentifier("one"),
+			newEndpointWithOwner("txt.oldformat2.test-zone.example.org", "\"heritage=external-dns,external-dns/owner=owner\"", endpoint.RecordTypeTXT, "").WithSetIdentifier("one"),
+			newEndpointWithOwner("oldformat2.test-zone.example.org", "bar2.loadbalancer.com", endpoint.RecordTypeA, "").WithSetIdentifier("two"),
+			newEndpointWithOwner("txt.oldformat2.test-zone.example.org", "\"heritage=external-dns,external-dns/owner=some-other-owner\"", endpoint.RecordTypeTXT, "").WithSetIdentifier("two"),
```

Comment:
`oldformat` and `oldformat2` were there to cover 2 different types: CNAME and A. What would you say about keeping `oldformat2` the way it was and add 4 new records which tests the setidentifier case? Similar to what `oldformat-otherowner` record is doing here - testing the case when the owner is different from the registry's one.

Reply:
Done